src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -100,7 +100,7 @@ export default function TokenTable({ setRowCount }: { setRowCount: (c: number) =
       <GridContainer>
         <HeaderRow />
         <TokenDataContainer>
-          {tokens.map(
+          {tokens?.map(
```

Comment:
This change should should not be necessary. It should be typed as non-nullable after L85.